From 6ce9bee9e23010893952a5b2f6e8fbeb6c4b3628 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Romain=20Tarti=C3=A8re?= <romain@blogreen.org>
Date: Mon, 25 Nov 2019 15:11:45 -1000
Subject: [PATCH] (maint) Add missing require: singleton

This file use the Singleton pattern implemented by Ruby's Singleton
class in the singleton module.  Require the singleton module before
using the Singleton class.
---
 lib/puppet/runtime.rb | 1 +
 1 file changed, 1 insertion(+)

diff --git lib/puppet/runtime.rb lib/puppet/runtime.rb
index 1d52cec59b..73bd9eb07f 100644
--- lib/puppet/runtime.rb
+++ lib/puppet/runtime.rb
@@ -1,4 +1,5 @@
 require 'puppet/http'
+require 'singleton'
 
 class Puppet::Runtime
   include Singleton
-- 
2.24.0

